content/docs/command-reference/freeze.md

```diff
@@ -1,32 +1,32 @@
 # freeze
 
-Freeze a [stage](/doc/command-reference/run). Use `dvc unfreeze` to unfreeze the
-stage.
+Fix the status of one or more [stages](/doc/command-reference/run) to _not
```

Comment:
Can we do something simpler? Frozen stage is never executed on `dvc repro` no matter it has changes dependencies or not.

Reply:
I didn't want to use the word "freeze" to explain `freeze` but OK. I like the sentence you suggested but it doesn't say what the command does. Maybe start with "Freeze a stage. Frozen stages..."

Reply:
Updated in cca7692d. PTAL